requirements/tests.txt

```diff
@@ -1,4 +1,5 @@
 coverage
 lmdb
 parameterized
+pre-commit
```

Comment:
We should not add `pre-commit` requirements here

Reply:
It's gone again. Please consider adding it somewhere else, where you find it more appropriate.